server/src/utils/functions.utils.ts

```diff
@@ -0,0 +1,4 @@
+import {randomInt} from 'crypto';
+export function generateRandomDigitsCode(length: number): string {
```

Comment:
please make spaces - 
```suggestion
import { randomInt } from 'crypto';
export function generateRandomDigitsCode(length: number): string {
```